integrated_channels/canvas/transmitters/learner_data.py

```diff
@@ -54,6 +54,6 @@ def assessment_level_transmit(self, exporter, **kwargs):
             payload: The learner completion data payload to send to Canvas
         """
         kwargs['app_label'] = 'canvas'
-        kwargs['model_name'] = 'CanvasLearnerDataTransmissionAudit'
```

Comment:
is this comment on this line out of date/ needs change? https://github.com/edx/edx-enterprise/blob/5738b989ff90ec093b422e6b63694853cd7a4092/integrated_channels/canvas/exporters/learner_data.py#L72

Reply:
that's a good find! I will update the doc 👍